source/documentation/managing_apps/ssh.erb

```diff
@@ -2,104 +2,98 @@
+You do not need to generate any SSH keys. The Cloud Foundry CLI handles authentication for you.
 
-### Connecting with multiple instances
+### Connecting to multiple instances
 
-You may be running multiple instances of an app (created using `cf scale` or ``instances:`` in the manifest).
+If you are running [multiple instances](/managing_apps.html#scaling) of an app, you can connect to a specific app instance using SSH.
 
-In this situation, each instance has a numerical instance index to distinguish it. You can see this in this example of the output of ``cf app exampleapp``:
+Each instance of an app has an attached number.
 
-```
-requested state: started
-instances: 3/3
-usage: 64M x 3 instances
-urls: exampleapp.london.cloudapps.digital
-last uploaded: Wed Dec 21 13:56:24 UTC 2016
-stack: cflinuxfs3
-buildpack: staticfile_buildpack
-
-.   state     since                    cpu    memory        disk         details
-0   running   2016-12-21 02:27:11 PM   3.0%   7.1M of 64M   6.8M of 1G
-1   running   2016-12-21 02:44:46 PM   1.0%   3.5M of 64M   6.8M of 1G
-2   running   2016-12-21 02:44:46 PM   1.0%   3.5M of 64M   6.8M of 1G
-```
-This example assumes that `exampleapp` is hosted in the [London region](orgs_spaces_users.html#regions).
-
-There are 3 instances, with instance indexes from 0 to 2.
-
-If you have multiple instances like this and use `cf-ssh`, you will be connected to the app with an instance index of 0.
-
-You can connect to a particular instance. For example, if you want to connect to instance 2, you can do this:
-
-```
-cf ssh --app-instance-index 2
-```
+1. Run `cf app APPNAME` to see the app instance numbers in the output. For example:
 
-### Creating TCP tunnels with SSH
-
-You can use the [Conduit plugin](/guidance.html#conduit) to connect your local system to your remote backing service instance. If you instead want to manually create a TCP tunnel with SSH, refer to this section.
+    ```
+    requested state: started
+    instances: 3/3
+    usage: 64M x 3 instances
+    urls: APPNAME.london.cloudapps.digital
+    last uploaded: Wed Dec 21 13:56:24 UTC 2016
+    stack: cflinuxfs3
+    buildpack: staticfile_buildpack
+
+    .   state     since                    cpu    memory        disk         details
+    0   running   2016-12-21 02:27:11 PM   3.0%   7.1M of 64M   6.8M of 1G
+    1   running   2016-12-21 02:44:46 PM   1.0%   3.5M of 64M   6.8M of 1G
+    2   running   2016-12-21 02:44:46 PM   1.0%   3.5M of 64M   6.8M of 1G
+    ```
 
-The `cf ssh` command supports [local port forwarding](https://en.wikipedia.org/wiki/Port_forwarding#Local_port_forwarding), which allows you to create tunnels from your local system to the application instance container. This is useful when you want to connect from your local system to a backing service that is only accessible from an app running on GOV.UK PaaS.
+1. Run `cf ssh --app-instance-index APP_INSTANCE_NUMBER` to connect to a specific instance.
```

Comment:
If the long form of the argument a style choice? If not ,`-i` is a more readable choice IMO

Reply:
Also, we're missing the APPNAME argument from the end of that command.

`cf ssh --app-instance-index APP_INSTANCE_NUMBER APPNAME`

Reply:
agree